docgen/src/guides/advanced-topics.md

```diff
@@ -153,3 +153,127 @@ class App extends Component {
 export default withRouter(App);
 ```
 
+## React Native
```

Comment:
Given the length of the topic, I propose (after this PR) that we rename advanced section into tutorials and give them their own pages. What do you think?